README.md

```diff
@@ -162,7 +157,7 @@ To extend the instrumentation with the OpenTelemetry Instrumentation for Java,
 you have to use a compatible API version.
 
 <!-- IMPORTANT: do not change comments or break those lines below -->
-The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.14.2<!--SPLUNK_VERSION--> is compatible
+The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.15.0<!--SPLUNK_VERSION--> is compatible
 with:
 
 * OpenTelemetry API version <!--OTEL_VERSION-->1.16.0<!--OTEL_VERSION-->
```

Comment:
Change the opentelemetry api and implementation versions, also change the versions that are used to render the badges at the top of the readme. Previous release didn't do these changes, check the diff from the pre release changes from an earlier release

Reply:
Done.